extensions/amp-social-share/1.0/test/test-amp-social-share.js

```diff
@@ -226,6 +229,49 @@ describes.realWin(
       ).to.be.equal('inherit');
     });
 
+    it('should show default and custom focus indicator styles', async () => {
+      element = win.document.createElement('amp-social-share');
+      element.setAttribute('type', 'email');
+      win.document.body.appendChild(element);
+      await waitForRender();
+
+      // element is not focused and does not have focus indication styles
+      expect(win.document.activeElement).to.not.equal(element);
+      expect(win.getComputedStyle(element)['outline']).to.equal(
+        'rgb(255, 255, 255) none 0px'
+      );
+      expect(win.getComputedStyle(element)['outlineOffset']).to.equal('0px');
+
+      // focus the button within the shadow DOM
+      const button = element.shadowRoot.querySelector("[part='button']");
```

Comment:
Here, we should be expecting `element.focus()` to work correctly.

Reply:
Added a test with `element.focus()`